### DISCUSSION TITLE
Getting "WebDriver.__init__() got multiple values for argument 'options'" while invoking locally

### DISCUSSION THREAD

**Discussion by ramanwaliagithub** [CloudBytes-Academy/CloudBytes.dev#179] [Q&A]
Getting the following error
chrome = webdriver.Chrome("/opt/chromedriver", options=chrome_options)options' {"errorMessage": "WebDriver.__init__() got multiple values for argument 'options'", "errorType": "TypeError", "requestId": "3*********ec8", "stackTrace": ["  File \"/var/task/app.py\", line 27, in handler\n    chrome = webdriver.Chrome(\"/opt/chromedriver\", options=chrome_options)\n"]}END

**Comment by rehanhaider** [comment]
> Getting the following error chrome = webdriver.Chrome("/opt/chromedriver", options=chrome_options)options' {"errorMessage": "WebDriver.**init**() got multiple values for argument 'options'", "errorType": "TypeError", "requestId": "3*********ec8", "stackTrace": [" File "/var/task/app.py", line 27, in handler\n chrome = webdriver.Chrome("/opt/chromedriver", options=chrome_options)\n"]}END

could you be a bit specific about which guide you were following?
